Demo/VIper-v2-Demo/VIper-v2-Demo/Application/Initializers/AlamofireInitializer.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  AlamofireInitializer.swift
+//  Pokedex
+//
+//  Created by Filip Beć on 27/04/2017.
+//  Copyright © 2017 Filip Beć. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import AlamofireNetworkActivityIndicator
+
+class AlamofireInitializer: NSObject, Initializable {
+
+    @objc func performInitialization() {
```

Comment:
I see no reason why this function should be `@objc` nor why this class should be `NSObject`

Reply:
Same applies to all other initializers, and some more classes / extension.